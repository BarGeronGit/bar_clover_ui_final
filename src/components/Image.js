import React from 'react'
import styled from 'styled-components'
const GameImage = styled.img`
  width: 200px;
  min-width: 200px;
  height: 200px;
  min-height: 200px;
  border-radius: 100%;
  object-fit: cover;
  border: 5px solid ${props =>props.palette};
  mix-blend-mode: ${props => props.blendMode || "unset"};
  object-position: ${props => props.position || "initial"};
`;

export const CloverImage = (props) => (
  <GameImage blendMode={props.blendMode} position={props.position} src={props.slotImg} alt={props.imgAlt} palette={props.palette} />
);
