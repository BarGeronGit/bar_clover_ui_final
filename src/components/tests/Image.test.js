import React from 'react';
import {CloverImage} from '../Image';
import renderer from 'react-test-renderer';

describe('Image Styled Components', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<CloverImage></CloverImage>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })});
