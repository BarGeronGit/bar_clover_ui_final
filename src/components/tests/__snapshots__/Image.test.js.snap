// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Styled Components renders correctly 1`] = `
<img
  alt={undefined}
  className="sc-bdVaJa iVBFAr"
  src={undefined}
/>
`;
